tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
sorry, it seems I didn't express myself right (you can blame the language barrier), I didn't mean that ffmpeg configure defaults changd. I meant whether building with --enable-vdpau / vaapi silently succeeds without va/vdpau headers present. that's definitely not the case for both va/vdpau in 3.3. so --enable-vaapi should fail on rbpi as well, you should move it to the ifneq block